robotoff/app/api.py

```diff
@@ -1055,6 +1061,45 @@ def on_get(self, req: falcon.Request, resp: falcon.Response, username: str):
         resp.media = {"count": {"annotations": annotation_count}}
 
 
+class DisplayInsightPredictionForProducts:
```

Comment:
Oh by the way, you should not call it `DisplayInsightPredictionForProducts` but just `PredictionCollection`, to be same as `InsightCollection`